src/components/button-group/examples/button-group-composite.tsx

```diff
@@ -51,22 +51,24 @@ export class ButtonCompositeExample {
     public render() {
         return [
             <limel-button-group {...this.props} onChange={this.handleChange} />,
-            this.renderForm(),
             <limel-example-event-printer
                 ref={(el) => (this.eventPrinter = el)}
             />,
+            this.renderForm(),
```

Comment:
Also, this commit says `docs(banner)`, but it should say `docs(button-group)` 🙂

Reply:
good catch ⚡